### PR TITLE
`Development`: Shorten e2e test duration

### PR DESCRIPTION
**Pull request by TheZoker**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

E2E tests take about 85 minutes and we want to reduce that run time

### Description

This PR reduces the runtime of the E2E test by almost 20 minutes by changing the following:
- Use shorter text inputs for most text exercises and communication tests
- Input the programming exercise code directly instead of typing it (this will also add support for other programming languages in the future 🎉)

#### Code Review
- [ ] Review 1
- [ ] Review 2

### CODE REVIEW

File: src/test/cypress/support/pageobjects/exercises/programming/OnlineEditorPage.ts [hunk 1]
```diff
@@ -200,6 +185,7 @@ export class OnlineEditorPage {
 export class ProgrammingExerciseSubmission {
     files: ProgrammingExerciseFile[];
     expectedResult: string;
+    packageName: string;
```
Comment (4ludwig4) [inline src/test/cypress/support/pageobjects/exercises/programming/OnlineEditorPage.ts:188]: There are several changes concerning the `packageName`. I needed more time to read it all in depth and am not really into Cypress Tests. What is being changed concerning those in general in this PR?

File: src/test/cypress/fixtures/exercise/programming/partially_successful/SortStrategy.txt
Comment (4ludwig4) [inline src/test/cypress/fixtures/exercise/programming/partially_successful/SortStrategy.txt:1]: Is this moved somewhere else or why is it not needed anymore? Is this related to the `packageName`?

File: src/test/cypress/support/pageobjects/exercises/programming/OnlineEditorPage.ts (hunk 1 above)
Comment (TheZoker) [reply src/test/cypress/support/pageobjects/exercises/programming/OnlineEditorPage.ts:188]: I moved the definition of the `packageName` from the cypress test itself to the submissions file.

File: src/test/cypress/fixtures/exercise/programming/partially_successful/SortStrategy.txt
Comment (TheZoker) [reply src/test/cypress/fixtures/exercise/programming/partially_successful/SortStrategy.txt:1]: The file was just not used, so I removed it

**Comment by 4ludwig4** [comment]
Code looks good to me, Tests are indeed about 15-20 minutes shorter. I guess you'll resolve the conflicts then no problem from my side to approve it!
